packages/react-core/src/components/NotificationBadge/NotificationBadge.tsx

```diff
@@ -28,16 +28,13 @@ export interface NotificationBadgeProps extends Omit<ButtonProps, 'variant'> {
    * notification badge.
    */
   isExpanded?: boolean;
-  /** @deprecated Use the variant prop instead - Adds styling to the notification badge to indicate it has been read. */
-  isRead?: boolean;
   /** Determines the variant of the notification badge. */
   variant?: NotificationBadgeVariant | 'read' | 'unread' | 'attention';
 }
 
 export const NotificationBadge: React.FunctionComponent<NotificationBadgeProps> = ({
-  isRead,
   children,
-  variant = isRead ? 'read' : 'unread',
+  variant,
```

Comment:
hmmm the default in what sense?

Reply:
@mmenestr In terms of the variant `read`, `unread` or `attention`
https://www.patternfly.org/v4/components/notification-badge#basic

Reply:
@tlabaj @mmenestr default and read do look the same. The core docs do say to specify a modifier for the state, though the requirement to add a state may have been a requirement from the previous designs, and looking at the code, it looks like the read modifier and default (no modifier) are the same more by chance, not because it's coded that way on purpose. A read notification badge is basically just an icon.

Unless we want to potentially introduce a "default" (no modifier) state for the notification badge that's different from "read", I think either updating the styling to be read by default (we would remove `.pf-m-read`) or assigning a default value for the prop (like the [button variant prop](https://github.com/patternfly/patternfly-react/blob/b00b0905ba45f4832fe8ae6ae19d47ea0d26e059/packages/react-core/src/components/Button/Button.tsx#L107), since a button also requires a modifier) would be fine.

A default/no-modifier badge could be something like
* neither read or unread, maybe you haven't clicked on it, or there is no state change regardless what you do to it
* we want a default state to be the regular icon button color (color-200) and the "read" color to be more grey, and/or italic (if there is text) or something